index.html

```diff
@@ -1,11 +1,15 @@
 <html>
   <head>
     <link rel="stylesheet" type="text/css" href="app.css" />
-    <script src="app.js"></script>
+  	<script src="app.js"></script>
```

Comment:
Watch your whitespace, this is a tab :) I Iike 2 spaces for HTML, and 2 or 4 spaces for JS.